### ISSUE TITLE
Create content with links

### ISSUE THREAD

**Issue by yantze** [guigrpa/docx-templates#20]
Hi,

How do I create a title with a link such as '+++=[title](http://xxx.com)+++' ?

Thanks.
yantze

**Comment by guigrpa** [comment]
Do you mean a link to a dynamic address, depending on the input data?

**Comment by vdechef** [comment]
I would also like to see this feature in doc-templates : a command like `+++LINK displayed text | http://my_site.com/endpoint.html +++`

**Comment by yantze** [comment]
@guigrpa Yes，just like @vdechef 's example.

**Comment by guigrpa** [comment]
Shipped in v2.6.0. Please check this version — note that the API is not exactly as you suggest, but rather more in line with the `IMAGE` command. Call it with a JS snippet that returns an object of the form: 

```js
{
  url: string,
  label?: string, // optional
}
```

Keep me posted if you like it, or if you find a problem with the new feature!

**Comment by guigrpa** [comment]
Examples (from the unit tests):

```
+++LINK ({ url: 'http://www.apple.com', label: 'Apple' })+++
+++LINK ({ url: 'http://www.apple.com' })+++
```
